src/middleware/validation.middleware.ts

```diff
@@ -82,7 +84,10 @@ export function checkValidations(req: Request, res: Response, next: NextFunction
           relevant_period: relevantPeriod,
           errors,
           FEATURE_FLAG_ENABLE_REDIS_REMOVAL: true,
-          activeSubmissionBasePath: getUrlWithParamsToPath(config.ACTIVE_SUBMISSION_BASE_PATH, req)
+          activeSubmissionBasePath: getUrlWithParamsToPath(config.ACTIVE_SUBMISSION_BASE_PATH, req),
+          pageParams: {
+            noChangeFlag
```

Comment:
This middleware is used in many requests, so if we start trying to solve all the possibilities, we may introduce other bugs.
In reality, validation should not be in a middleware but in the use case, which would allow errors to be managed independently.

Reply:
Yep, agree with all that and that we shouldn't randomly throw these params in now too _just in case_.

Just something to remember for the future. Will resolve this and approve.